src/schema/v2/homeView/sections/ShowsForYou.ts

```diff
@@ -10,4 +17,24 @@ export const ShowsForYou: HomeViewSection = {
     title: "Shows for You",
   },
   requiresAuthentication: true,
+
+  shouldBeDisplayed: (_context: ResolverContext) => true,
+
+  resolver: withHomeViewTimeout(async (parent, args, context, info) => {
+    if (!ShowsConnection.resolve) return emptyConnection
+
+    const finalArgs = {
+      ...args,
+      status: EventStatusEnums.getValue("RUNNING_AND_UPCOMING")?.value,
+    }
+
+    const result = await ShowsConnection.resolve(
+      parent,
+      finalArgs,
+      context,
+      info
+    )
```

Comment:
**suggestion / non-blocking**: I know we use this approach elsewhere, mainly to avoid duplicating the resolver logic, but I wonder how much benefit we're really getting from reusing another field's resolver.

In this specific case the [me/showsConnection resolve function](https://github.com/artsy/metaphysics/blob/fc6a57aa2d25cb157b68e5620665714a80324b18/src/schema/v2/me/showsConnection.ts#L41-L94):
- Exits early if user is logged out
- Translates input args to meShowsLoader data loader args
- Injects an IP-backed `near` fallback
- Invokes the data loader
- Returns a connection

This seems pretty standard and reasonable for this section to handle itself and we wouldn't be coupling the implementation here with how shows are currently resolved under me/showsConnection.

For more complicated resolver logic, we've had success in the past extracting functions that can be used across different field resolvers.